adopt-a-dog/styling/stylesheet.css

```diff
@@ -0,0 +1,392 @@
+}
+.dog-info{
+  width :12rem;
+  height : 23rem;
+  display: flex;
+  flex-direction: column;
+  align-items:flex-start;
+  justify-content: space-around;
+  padding-left: 10px;
+  margin-right: 10px;
+  margin-left: 10px;
+  margin-bottom: 1.8rem;
+  margin-top: 1.8rem;
+  border: 1px solid #e6e6e6;
+}
+.dog-img{
+width: 12rem;
+height: 7rem;
+}
+.dog-info > h3{
+  color: #003366;
+}
+.dog-info > h5{
+  font-size: 0.7rem;
+  font-family: 'Noto Sans SC', sans-serif;
+}
+.dog-info > p{
+  font-size: 0.7rem;
+  font-family: 'Noto Sans SC', sans-serif;
+}
+.checkout-button
+{
+  background-color: #66cc00;
+  color: white;
+  width: 8rem;
+  height: 1.5rem;
+  margin-left:  25px;
+  outline: none;
+  border: none;
+  padding: 3px;
+}
+.checkout-button
+{
+  background-color: #66cc00;
+  color: white;
+  width: 8rem;
+  height: 1.5rem;
+  margin-left:  25px;
+  outline: none;
+  border: none;
+  padding: 3px;
+  font-size: 1rem;
+}
+.grey-button
+{
+  background-color: grey;
+  color: white;
+  width:10rem;
+  padding: 5px;
+  border: none;
+  outline: none;
+  margin-left: auto;
+  margin-right: auto;
+  display: block;
+}
+/***** footer *****/
+footer{
+  width: 100%;
+  height: 25rem;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  background-color: #004466;
+  color: white;
+}
+.footer-wrapper{
+  width: 100%;
+  height: 18rem;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: space-around;
+}
+.logo-bottom{
+ width: 50%;
+ height: 30%;
+ max-width: 20rem;
+ min-width: 5rem;
+ align-self: center;
+ }
+.nav-footer-container{
+  width : 100%;
+}
+.nav-footer{
+  padding: 0.5rem;
+  width: 100%;
+  display: flex;
+  flex-direction: row;
+  justify-content:center;
+  list-style-type: none;
+}
+.nav-footer li::after{
+  padding-left: 20px;
+  padding-right: 20px;
+  content: "/";
+}
+.nav-footer li:last-child::after{
+  content: "";
+}
+.nav-footer li a{
+  text-decoration : none;
+  color: white;
+  font-size: 1rem;
+}
+.footer-text{
+  height: 3rem;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: space-between;
+}
+/* blog Web-pages */
+.blog-img{
+  width: 20rem;
+  height: 15rem;
+}
+.blog-topic{
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: space-around;
+  width: 100%;
+  margin: 0;
+  margin-top: 40px;
+  margin-bottom: 60px;
+}
+.blog-content{
+  display: flex;
+  flex-direction: column;
+  width: 60%;
+  padding-left: 30px;
+}
+.blog-content p{
+  font-size: 1rem;
+  align-self: center;
+  font-family: 'Noto Sans SC', sans-serif;
+}
+.blog-content h3{
+font-size: 1.6rem;
+color: #003366;
+margin-bottom: 20px;
+}
+/* Checkout Form */
+.checkout-form{
+  width: 60%;
+  height: 45rem;
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+  padding: 20px;
+}
+.form-multi-content{
+  width: 100%;
+  display: flex;
+  justify-content:space-between;
+}
+.form-multi-content-radio{
```

Comment:
see how you could leverage input[type="text"] and input[type="radio"] to layout your form a bit, you shouldn't need flexbox here.

Reply:
I removed the flexbox from form. Kindly let me know whether I did it correctly.